examples/heartbeat_caravel_wrapper/flows/mpwflow.py

```diff
@@ -0,0 +1,42 @@
+import siliconcompiler
+
+def make_docs():
+    '''An RTL2GDS flow for the eFabless MPW shuttle. This flow is based off the
+    basic serial asicflow, with the addition of fixdef and addvias steps for
+    doing MPW-specific fixes to the post-routing DEF and gate-level netlist,
+    respectively.'''
+    chip = siliconcompiler.Chip('<design>')
+    setup(chip)
+    chip.set('option', 'flow', 'mpwflow')
+    return chip
+
+def setup(chip):
+    flow = 'mpwflow'
+
+    flowpipe = [
+        ('import', 'surelog'),
+        ('syn', 'yosys'),
+        ('floorplan', 'openroad'),
+        ('place', 'openroad'),
+        ('cts', 'openroad'),
+        ('route', 'openroad'),
+        ('fixdef', 'fixdef'),
+        ('dfm', 'openroad'),
+        ('export', 'klayout')
+    ]
+
+    step, tool = flowpipe[0]
+    chip.node(flow, step, tool)
+
+    prevstep = step
+    for step, tool in flowpipe[1:]:
+        chip.node(flow, step, tool)
+        chip.edge(flow, prevstep, step)
+        prevstep = step
+
+    chip.node(flow, 'addvias', 'addvias')
+    chip.edge(flow, 'dfm', 'addvias')
```

Comment:
I like how this demonstrates the potential for parallel flowgraph steps; the netlist post-processing can happen in parallel with the GDS-streaming step, which this captures.